index.js

```diff
@@ -43,6 +43,10 @@ module.exports = {
         route = resolveRouteImport(`${globOptions.cwd}/${file}`);
         route = _.defaultsDeep(route, options.defaultRoute, defaultRoute);
 
+        if (options.pathPrefix) {
+          route.path = options.pathPrefix + route.path;
```

Comment:
Isn't there a way that route.path does not start with `/`. Just for security would be good to stitch this path together in smarter way with a function that adds slashes if not there.